packages/core/src/hooks/form/index.ts

```diff
@@ -159,7 +159,8 @@ export const useForm = <
     id,
     queryOptions: {
       // Only enable the query if it's not a create action and the `id` is defined
-      enabled: !isCreate && id !== undefined,
+      // AND the external enabled condition (if provided) is also true
+      enabled: (!isCreate && id !== undefined) && (props.queryOptions?.enabled ?? true),
       ...props.queryOptions,
```

Comment:
Doesn’t `queryOptions` always override enabled? Or did I understand wrong? I think it should be like this:

```suggestion
			...props.queryOptions,
      // AND the external enabled condition (if provided) is also true
      enabled: (!isCreate && id !== undefined) && (props.queryOptions?.enabled ?? true),
     
```